packages/fxa-content-server/server/lib/csp/blocking.js

```diff
@@ -37,6 +37,13 @@ module.exports = function (config) {
   const GOOGLE_AUTH = 'https://accounts.google.com';
   const APPLE_AUTH = 'https://appleid.apple.com';
 
+  const tracingConfig = config.get('tracing');
+  let TRACE_OTLP_URL = undefined;
+  if (tracingConfig.jaeger && tracingConfig.jaeger.otlpUrl) {
+    TRACE_OTLP_URL = url.parse(tracingConfig.jaeger.otlpUrl);
+    TRACE_OTLP_URL = TRACE_OTLP_URL.href.replace(TRACE_OTLP_URL.pathname, '');
```

Comment:
What is this about?

Reply:
This allows the cors request through. We already have a a setting which indicates the full URL of the tracing endpoint, so this just extracting the root of the URL of that end point to configure CORS settings. 

Maybe it'd be more clear to just have a separate config option for this?

Reply:
Oh, nah.  It was my bad; I commented before reading farther to understand how that was used.